academic_helper/logic/shnaton_parser.py

```diff
@@ -206,11 +200,8 @@ def parse_lecturers(lecturers):
 
     for lecturer in lecturers.contents:
         # TODO: This order is not right for course 1921!
-        if lecturer.string is not None:
+        if lecturer.string is not None and lecturer.string.strip() != '':
             ret.append(lecturer.string)
-        elif lecturer.name == "br" and lecturer.previous_sibling not in ret:
-            ret.append(ret[-1])
```

Comment:
Are you completely sure these lines can be removed?

Reply:
Not sure. Will be reverted

Reply:
reverted in the new commit